code-studio/src/js/pd/workshop_dashboard/components/workshop_table.jsx

```diff
@@ -0,0 +1,109 @@
+/* global React */
+
+var WorkshopTableRow = require('./workshop_table_row.jsx');
```

Comment:
nit: Just realized you've been using snake_case for your component files. Our preference elsewhere has been for CamelCase, though it's probably not worth going through and changing everything